index.js

```diff
@@ -1,11 +1,14 @@
 const { getOptions } = require('loader-utils');
 const postcss = require('postcss');
-const kremId = require('./src/kremling-id');
 const postcssKremling = require('./src/postcss-kremling-plugin');
 
+let globalId = 0;
+
 module.exports = function(source) {
+  const id = globalId;
+  globalId += 1;
   const loaderOptions = getOptions(this) || {};
-  let kremlingNamespace = 'data-kremling';
+  let kremlingNamespace = 'kremling';
```

Comment:
any time the `kremling` repo changes the default namespace we have to change it here too

not sure how else to handle this (besides making it required)

Reply:
👍 i think that's fine. I don't think we'll change the default namespace again, at least for the foreseeable future. The `data-` prefix is the only thing that i can think of that we would have wanted to change.